### ISSUE TITLE
history tracer plugin: browser shows no activity

### ISSUE THREAD

**Issue by maphew** [leo-editor/leo-editor#1503]
after solving #1502, I have a tracer that is recording history, but the browser pointed at leo-ver-serv is not showing any activity.

![image](https://user-images.githubusercontent.com/486200/75101856-4f2d9a00-5598-11ea-9886-2bbac0988492.png)

**Comment by maphew** [comment]
It's working for me now on Linux. Maybe it always did and the only thing missing was the port setting in Leo.

Still not working on my Win10 machine though. The console which I started leo-ver-serv from shows:
~~~
$ leo-ver-serv C:\Users\mhwilkie\.leo\.leoRecentFiles.txt 8088
post snapshot 5679 bytes at 2020-02-25T21:30:20.000000
post snapshot 6174 bytes at 2020-02-25T21:31:10.000000
post snapshot 6226 bytes at 2020-02-25T21:34:00.000000
~~~

Leo log pane:
~~~
Leo Log Window
Leo 6.2-b1-devel, devel branch, build adededb0cd
2020-02-25 04:15:38 -0600
Python 3.7.1, PyQt version 5.12.6
Windows 10 AMD64 (build 10.0.18362) SP0
read outline in 0.03 seconds
reading settings in D:/code-maphew/leo-editor/leo/plugins/leoPluginsRef.leo
snap 1 myLeoSettings.leo
snap 1 myLeoSettings.leo
wrote recent file: C:/Users/mhwilkie/.leo/.leoRecentFiles.txt
 saved: myLeoSettings.leo
snap 1 myLeoSettings.leo saved: myLeoSettings.leo
~~~

**Comment by vitalije** [comment]
I believe it is fixed in the latest 0.1.8 version of leo-ver-serv. Please reopen if you see the problem again.
